### ISSUE TITLE
[Android] Update toast notification background color in Rewards panel (Feb 2022)

### ISSUE THREAD

**Issue by Miyayes** [brave/brave-browser#21059]
Android version of https://github.com/brave/brave-browser/issues/21058. 

Applies to the background color of _all_ toast notifications, not just the one pictured above.

**Comment by Miyayes** [comment]
Closing issue as it is complete per https://github.com/brave/brave-core/pull/13500. (Should have closed automatically when this PR was merged...)

**Comment by kjozwiak** [comment]
Unfortunately this was never verified by QA as the milestone on the issue was never set so QA never saw/verified the above while running through the `1.41.x` milestone. Cleaning up https://github.com/orgs/brave/projects/40/views/1?filterQuery= right now hence running into this issue.
